src/ResourceManager/ManagementGroups/Commands.ManagementGroups/Commands.ManagementGroups.csproj

```diff
@@ -0,0 +1,197 @@
+      <HintPath>..\packages\Microsoft.Net.Http.2.2.22\lib\net45\System.Net.Http.Primitives.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="System.Net.Http.WebRequest" />
+    <Reference Include="System.Xml.Linq" />
+    <Reference Include="System.Data.DataSetExtensions" />
+    <Reference Include="Microsoft.CSharp" />
+    <Reference Include="System.Data" />
+    <Reference Include="System.Net.Http" />
+    <Reference Include="System.Xml" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="Cmdlets\NewAzureRmManagementGroup.cs" />
+    <Compile Include="Cmdlets\NewAzureRmManagementGroupSubscription.cs" />
+    <Compile Include="Cmdlets\RemoveAzureRmManagementGroup.cs" />
+    <Compile Include="Cmdlets\GetAzureRmManagementGroup.cs" />
+    <Compile Include="Cmdlets\RemoveAzureRmManagementGroupSubscription.cs" />
+    <Compile Include="Cmdlets\UpdateAzureRmManagementGroup.cs" />
+    <Compile Include="Common\AzureManagementGroupAutoRegisterCmdletBase.cs" />
+    <Compile Include="Common\AzurePrecheckRegistrationAndAutoRegisterDelegatingHandler.cs" />
+    <Compile Include="Common\AzureManagementGroupsCmdletBase.cs" />
+    <Compile Include="Common\AzurePrecheckRegistrationAndAutoRegisterCmdletBase.cs" />
+    <Compile Include="Common\Constants.cs" />
+    <Compile Include="Common\Utility.cs" />
+    <Compile Include="Models\PSManagementGroup.cs" />
+    <Compile Include="Models\PSManagementGroupChildInfo.cs" />
+    <Compile Include="Models\PSManagementGroupInfo.cs" />
+    <Compile Include="Models\PSManagementGroupNoChildren.cs" />
+    <Compile Include="Models\PSParentGroupInfo.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="Resource.Designer.cs">
+      <AutoGen>True</AutoGen>
+      <DesignTime>True</DesignTime>
+      <DependentUpon>Resource.resx</DependentUpon>
+    </Compile>
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\Common\Commands.Common.Authentication.Abstractions\Commands.Common.Authentication.Abstractions.csproj">
+      <Project>{70527617-7598-4aef-b5bd-db9186b8184b}</Project>
+      <Name>Commands.Common.Authentication.Abstractions</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\..\Common\Commands.Common\Commands.Common.csproj">
+      <Project>{5ee72c53-1720-4309-b54b-5fb79703195f}</Project>
+      <Name>Commands.Common</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\Common\Commands.ResourceManager.Common\Commands.ResourceManager.Common.csproj">
+      <Project>{3819d8a7-c62c-4c47-8ddd-0332d9ce1252}</Project>
+      <Name>Commands.ResourceManager.Common</Name>
+    </ProjectReference>
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="..\AzureRM.ManagementGroups.psd1">
+      <Link>AzureRM.ManagementGroups.psd1</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Include="app.config" />
```

Comment:
@rajshah11 please remove this reference to `app.config`

Reply:
Removed.